Missions render

frempongdev opened the pull request:
## In this PR, I made the following additions ➕ ➕ 🟢 

### - Implemented Missions Render inn Missions Route.
  - Used `useSelector()` Redux Hook to select the state slices and rendered lists of missions in Missions routes.

josuerushanika commented on src/components/Missions.js:
```diff
@@ -1,17 +1,41 @@
-// import Table from 'react-bootstrap/Table';
+import { useDispatch, useSelector } from 'react-redux';
+import Table from 'react-bootstrap/Table';
 import { useEffect } from 'react';
+import { Badge } from 'react-bootstrap';
 import { fetchMissions } from '../redux/mission/missionSlice';
 
```
![image](https://user-images.githubusercontent.com/100031798/227005283-e00cadcc-f157-4237-8dbb-bf0bf38cbc24.png)

frempongdev replied:
Hi @josuerushanika 
Thank You for addressing the issues above. It appears that the project that does run in the browser as described . However i would  ask that you install the following **React Bootsrap** dependencies and try running the program again.

If the issues persist afterwards, kinfly draw my attention so i find another solution since it runs without flaws here.

Kindly follow [this link](https://react-bootstrap.github.io/getting-started/introduction) to install React Boostrap

josuerushanika reviewed (changes requested):
Hello @frempongdev  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![Almost there](https://media.giphy.com/media/Rv6GGaNrRbtNConlYo/giphy.gif)

## Highlights
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
The application doesn't work in the broswer as expected please try to fix .


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

frempongdev commented:
Hi @josuerushanika 
Thank You for addressing the issues above. It appears that the project that does run in the browser as described . However i would  ask that you install the following **React Bootsrap** dependencies and try running the program again.

If the issues persist afterwards, kinfly draw my attention so i find another solution since it runs without flaws here.

Kindly follow [this link](https://react-bootstrap.github.io/getting-started/introduction) to install React Boostrap

josuerushanika reviewed (approved):
Hi @frempongdev ,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/wZjlCH43M3M0U/giphy.gif)



## Highlights
- Git flow followed 👌
- PR description is descriptive 📝

## Optional suggestions
[OPTIONAL] Kindly make sure you fix the merge conflict before submitting it for review.
<img width="888" alt="Screenshot 2023-02-07 at 9 43 58 PM" src="https://user-images.githubusercontent.com/25887253/217337202-3f2d450f-18ca-427e-be60-75acc99e7fc0.png">


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
